lib/griddler/mandrill/adapter.rb

```diff
@@ -59,7 +59,17 @@ def full_email(contact_info)
         end
       end
 
+      def add_base64_field(event, key, default_value)
+        if event[key]
+          event[key].each_value do |value| 
+            value[:base64] = default_value if !value.has_key?(:base64)
+          end
+        end
+      end
+
       def attachment_files(event)
+        add_base64_field(event, :images, true)
```

Comment:
There reason for that is separation knowledge about the images from files() method.